### PR TITLE
Add web socket events

### PR DESCRIPTION
**Pull request by irenesmith**
Adding BCD data for WebSocket events. [Sprints #1104](https://app.zenhub.com/workspaces/mdn-sprint-board-5ab3d23fd0f4ea53b0022a61/issues/mdn/sprints/1104)

Data is based on the existing BCD for the event handler properties.
Only WebSocket events are included in this PR.

### CODE REVIEW

File: api/WebSocket.json [hunk 1]
```diff
@@ -224,7 +224,62 @@
...
+              "version_added": null
+            },
+            "safari": {
+              "version_added": null
+            },
+            "safari_ios": {
+              "version_added": null
+            },
+            "webview_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/WebSocket.json]: This should be true. Please double-check the webview support for the other socket events using this list: https://cs.chromium.org/chromium/src/android_webview/tools/system_webview_shell/test/data/webexposed/global-interface-listing-expected.txt?g=0&l=7724

**Comment by Elchi3** [comment]
@irenesmith can you please follow-up on this PR? Let me know if I can help with anything.

File: api/WebSocket.json (hunk 1 above)
Comment (irenesmith) [reply api/WebSocket.json]: I think I've appropriately fixed this problem now.

**Comment by irenesmith** [comment]
> 
> 
> Compat data for events should get the identifier `eventname_event` and a description `<code>eventname</code>` event. See for example https://github.com/mdn/browser-compat-data/blob/master/api/MediaStream.json#L155-L206.
> 
> This is needed because otherwise event names can clash with property or method names.
> 
> In this PR that convention is not followed, so the entry here for the `close` event is overwriting the entry for the [`close()`](https://developer.mozilla.org/en-US/docs/Web/API/WebSocket/close) method.
> 
> Also as far as I can tell the data itself is not being taken from the corresponding `on-` event handler properties, but is just a clone of the data for the `close()` method (see for example the stuff about support for parameters in Firefox).

I realized that and have fixed the method.

**Comment by irenesmith** [comment]
I have also gone through and fixed the descriptions.

**Comment by Elchi3** [comment]
The descriptions are meant to be in this format: 
`"description": "<code>close</code> event",`
and not contain a longer sentence.

**Comment by irenesmith** [comment]
> 
> 
> Compat data for events should get the identifier `eventname_event` and a description `<code>eventname</code>` event. See for example https://github.com/mdn/browser-compat-data/blob/master/api/MediaStream.json#L155-L206.
> 
> This is needed because otherwise event names can clash with property or method names.
> 
> In this PR that convention is not followed, so the entry here for the `close` event is overwriting the entry for the [`close()`](https://developer.mozilla.org/en-US/docs/Web/API/WebSocket/close) method.
> 
> Also as far as I can tell the data itself is not being taken from the corresponding `on-` event handler properties, but is just a clone of the data for the `close()` method (see for example the stuff about support for parameters in Firefox).

I have fixed the names for the events and made sure that the correct version information is included.

**Pull request merged**